package.json

```diff
@@ -63,7 +63,7 @@
     "@babel/plugin-transform-react-jsx": "^7.14.3",
     "@bpmn-io/element-template-chooser": "^0.0.5",
     "@bpmn-io/element-templates-icons-renderer": "^0.1.2",
-    "@bpmn-io/properties-panel": "^0.17.0",
+    "@bpmn-io/properties-panel": "^0.18.0",
```

Comment:
We need to update the peer dependencies as well, to avoid stuff like https://github.com/bpmn-io/internal-docs/issues/599, right?

Reply:
Good catch, I updated it 👍